docs/protocol/non-evm-implementations.mdx

```diff
@@ -0,0 +1,23 @@
+While our public documentation currently focuses on EVM compatibility, we offer varying levels of support for additional ecosystems. These ecosystems may not have full feature parity or complete tooling integration (such as TypeScript CLI support) at this time. We're actively working to expand our documentation and tooling across these platforms. If you're excited about building on top of these or bringing Hyperlane to other new ecosystems reach out on [our Discord](https://discord.gg/hyperlane)!
+
+# CosmWasm
+
+See the CosmWasm implementation of Hyperlane [here](https://github.com/many-things/cw-hyperlane), and follow [this guide](https://github.com/many-things/cw-hyperlane/blob/main/DEPLOYMENT.md) on how to deploy to a Cosmos appchain here. We're live on Celestia, Stride and Neutron as of July 2024.
+
+# Sealevel
+
+Hyperlane supports Solana Virtual Machine (SVM), and is currently live supporting Zebec's [Nautilus Bridge](https://medium.com/hyperlane/case-study-interoperability-for-the-modular-rollup-stack-featuring-nautilus-d01118fe31ec). In addition, it is live on [Eclipse testnet](https://medium.com/hyperlane/hyperlane-expands-to-solana-vm-in-collaboration-with-eclipse-8465616fd0e4).
+
+You can find the [Sealevel contract implementation here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/sealevel/programs/mailbox/src/processor.rs).
```

Comment:
while we're at it imo it's fine to just point to `https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/sealevel/programs`